### ISSUE TITLE
Phantom Weight Tallied in Total Weight Carried Field

### ISSUE THREAD

**Issue by chaoscommencer** [Heliomance/HeroForge-Anew#45]
The total weight in the character sheet appears to be tallying up non-zero default values from extra hidden columns.  It tacked on about 30-40 lbs of extra weight to my champion (who has no armor and has attacks with weapons that total around 6 lbs--in case the weight of the weapons in the attacks is intended to be included in the total.  I'm not sure what those hidden values are supposed to represent, but I know my lv 1 wizard with nothing extra should not have whatever hidden weight that was.  Changing the formula in the Total Weight box to just sum the two visible columns solved the problem for me, but whether that formula change is  actually what was intended in the first place or whether someone accidentally pushed some invalid default settings, I don't know.

Potential patch:
Formula change for the TotalWeightCarried box:
=SUM(Z28:AC62,BA28:BD62)+IF(V15="",0,V15)+IF(AI20="",0,AI20)-ExpertPackerBonus
->
=SUM(Z28:Z62,BA28:BA62)+IF(V15="",0,V15)+IF(AI20="",0,AI20)-ExpertPackerBonus

I imagine the actual problem lies in the calculations being done in those hidden cells, but I'm not sure what they're for.  Perhaps someone more familiar with it will be able to determine that faster.

**Comment by TwistedFaith84** [comment]
Those "Hidden" cells from the formula aren't actually hidden. If you change the VIEW to show the Headings on Character Sheet II you'll notice that those "Hidden" cells are just compressed rows and columns.
The formula =SUM(Z28:AC62,BA28:BD62)+IF(V15="",0,V15)+IF(AI20="",0,AI20)-ExpertPackerBonus is correct. Although if I remember that actually isn't including the equipped magic items weight. You could be carrying 30-40 lbs from someplace else or maybe your character might has messed up. Try  remaking it on a new sheet and see if that helps.

**Comment by chaoscommencer** [comment]
Looked into this a little more.  The existing formula is unnecessarily including additional cells that have been merged with the Z and BA columns to provide easier to read cells.  Theoretically those extra cells should have no value, but it appears parsing some of those merged cells returns the same value as in the Z and BA columns when read.  This is why, as I've discovered, the weight I am seeing is actually triple my actual weight.  Also the example I am testing uses no armor nor shield (where the other values in the formula are coming from).  It is a plain level 1 wizard with only some equipment and about 6 lb of weapons in the attacks section.
Looks like an issue with whatever settings can be set when merging (whether the value is replicated across all merged cells or exists only in the first cell).  Some of the cells in replicate their values in all four cells, but some of them only add the weight entered to the first cell.  This is why instead of being x4 it is closer to x3 in my case (and was a little more curious to debug).

The proposed change to the formula is the correct fix for this bug, as it is less work for Excel to sum one column than four.  Another fix which could be included would be to standardize the formatting for merged cells (at least local to the Equipment section).  I would expect if this error shows up here it probably exists elsewhere throughout the spreadsheet as well, but finding the other locations may be more difficult.
